### ISSUE TITLE
Feature request: Allow uploading of .json or .txt files containing json formatted data (for dealing with huge data sets)

### ISSUE THREAD

**Issue by GOESPROTOCALL** [konklone/json#235]

**Comment by konklone** [comment]
Yes, I would like to allow people to drag and drop files onto the browser instead of pasting it as text. Any interest in providing a PR for this?

**Comment by GOESPROTOCALL** [comment]
Sorry, but what do you mean by PR? Public relations?

**Comment by konklone** [comment]
Pull request - it's a GitHub feature where you can submit the code changes
yourself for my review.

>

**Comment by GOESPROTOCALL** [comment]
Nah, I'm not much more than a skiddy so I probably won't be able to do the mods myself successfully.

**Comment by bashmail** [comment]
I am also seeking for this feature
